include/highfive/bits/H5PropertyList_misc.hpp

```diff
@@ -347,6 +347,16 @@ inline void Shuffle::apply(const hid_t hid) const {
     detail::h5p_set_shuffle(hid);
 }
 
+inline Blosc::Blosc(unsigned int level, unsigned int shuffle, unsigned int compressor) {
+    _cd_values[4] = level;
+    _cd_values[5] = shuffle;
+    _cd_values[6] = compressor;
+}
+
+inline void Blosc::apply(const hid_t hid) const {
+    detail::h5p_set_blosc(hid, _cd_values);
```

Comment:
This function likely can't be called twice. Therefore, if two property lists with Blosc are created I suspect a misleading error message will be printed. It's unclear if it'll work, because `register_blosc` seems to unconditionally `return 1;` (despite defining and setting a `retval`).